Gemfile

```diff
@@ -69,6 +69,7 @@ end
 group :test do
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/223335176-b21ba63e-9720-43bc-ae6e-49d7f3f1c074.png)

- [ ] It would be nice if you make add the following commands in their respective section to make your readme file descriptive enough :blush:

### Install
```
cd Blog-App
bundle install
rails db:create
rails db:migrate
```
### Run tests
```
rspec spec
```